src/paths/all_shortest_paths.c

```diff
@@ -32,8 +32,8 @@
 #include <string.h>  /* memset */
 
 /**
- * \function igraph_get_all_shortest_paths
- * \brief All shortest paths (geodesics) from a vertex.
+ * \function igraph_get_all_shortest_paths_cutoff
+ * \brief All shortest paths (geodesics) from a vertex, with cutoff.
```

Comment:
Let's add `\experimental` for all new functions, even if changes are unlikely. We can remove that in 0.11. `igraph_distances_cutoff()` is also marked as experimental.

Reply:
These docs say "added in 0.2", but that's for `igraph_get_all_shortest_paths()`, not for `igraph_get_all_shortest_paths_cutoff()`.